helpers/lib/cdnify.js

```diff
@@ -26,28 +24,38 @@ module.exports = globals => {
             return '';
         }
 
-        if (/^(?:https?:)?\/\//.test(path)) {
+        if (path.startsWith('http') || path.startsWith('//')) {
```

Comment:
I think to be safe with the refactor, we should be more explicit here:

```
if (path.startsWith('https://') || path.startsWith('http://') || path.startsWith('//'))
```

Reply:
done.